code/datums/components/sizzle.dm

```diff
@@ -0,0 +1,22 @@
+/datum/component/sizzle
+	var/mutable_appearance/sizzleicon
+	var/sizzlealpha = 0
+	dupe_mode = COMPONENT_DUPE_UNIQUE_PASSARGS
+
+/datum/component/sizzle/Initialize()
+	if(!isatom(parent))
+		return COMPONENT_INCOMPATIBLE
+	sizzlealpha += 5
```

Comment:
Bad. Set sizzlealpha = 5 in the definition and use the InheritComponent proc to increase the amount of alpha. 
Initialize is NOT called on this datum when you add a duplicate.

Reply:
do i change the dupe_mode

Reply:
No (although since you don't use any args, you could just COMPONENT_DUPE_UNIQUE instead)

You just put sizzlealpha into the InheritComponent proc instead of in Initialize because that's just the way it works.